suseconnect-ng.spec

```diff
@@ -46,8 +46,8 @@ BuildRequires:  s390-tools
 BuildRequires:  systemd
 %endif
 
-Obsoletes:      SUSEConnect < 0.3.99
-Provides:       SUSEConnect = 0.3.99
+Obsoletes:      SUSEConnect < 1.0.0
+Provides:       SUSEConnect = 1.0.0
```

Comment:
if you plan to tag suseconnect-ng as version 1.0.0 why not use `%{version}` here instead of explicit values?

Reply:
Because the version of suseconnect-ng might keep growing, but suseconnect ruby will never surpass `1.0.0`. I prefer being explicit on this.

Reply:
sounds good